### DISCUSSION TITLE
"Note" section of the react-inlinesvg README has broken links

### DISCUSSION THREAD

**Discussion by beausmith** [gilbarbara/react-joyride#686] [General]
Hello 😄 

The "Note" section of the react-inlinesvg README has broken links which I (and others) would like to read more about:
https://github.com/gilbarbara/react-inlinesvg#note

**Comment by gilbarbara** [comment]
hey @beausmith 

Why are you posting about react-inlinesvg in this repo? Open an issue there...

**Comment by beausmith** [comment]
When I follow the steps to post an issue there, it redirected me here.

**Reply by beausmith** [reply]
![image](https://user-images.githubusercontent.com/28444/118036644-acd70f00-b321-11eb-973a-fc2267905427.png)
 
The "open" link directed me to this forum.

**Reply by gilbarbara** [reply]
ok, I'll fix that.
But you can create a bug

**Comment by beausmith** [comment]
Done: https://github.com/gilbarbara/react-inlinesvg/issues/169
